app/src/app/[lng]/settings/page.tsx

```diff
@@ -51,6 +48,26 @@ export default function Settings({
 
   const { t } = useTranslation(lng, "settings");
 
+  const { data: userInfo, isLoading: isUserInfoLoading } =
+    api.useGetUserInfoQuery();
+
+  const { data: currentUserData, isLoading: isUserDataLoading } =
+    api.useGetUserQuery(
+      {
+        locode: userInfo?.defaultCityLocode!,
+        userId: userInfo?.userId!,
+      },
+      { skip: !userInfo },
+    );
```

Comment:
What is this query for? Can't you get the same data from the other two requests?

We should limit the amount of API requests to the absolute minimum for a snappy user experience.